### ISSUE TITLE
Join-server cannot synchronize articles with existing Android clients

### ISSUE THREAD

**Issue by JulyLies** [laurent22/joplin#8362]
I updated my Joplin-server, but forgot to migrate the data in docker during the update. Unfortunately, I did not save the data, so it is now effectively a new Joplin-server. Although I configured the same password, the notes that were previously on the server cannot be synced to the new server. Only the newly created notes can be synced when Joplin on my mobile device is connected.

## Environment
Docker Joplin-Server : 2.11.1
Android System：11
Android Joplin:2.9.8

**Comment by laurent22** [comment]
If you don't have a backup of your server data, you can export a JEX file from the desktop app, then reimport it and sync

**Comment by JulyLies** [comment]
> If you don't have a backup of your server data, you can export a JEX file from the desktop app, then reimport it and sync

Sorry, I didn't express my issue clearly. My desktop Joplin and the Joplin Server I built have no data left, only the previous Joplin notes on my Android phone. I rebuilt the Joplin Server, but the old notes on my phone couldn't be synchronized to my Joplin Server, but the newly created notes could be synchronized. I couldn't find a solution on forums or Google.
